### DISCUSSION TITLE
Mac statuskit-cloud.db

### DISCUSSION THREAD

**Discussion by macuser321123** [sqlitebrowser/sqlitebrowser#2920] [Q&A]
When browsing the Library folder on Mac (**~/Library/StatusKit/database**),

I'm trying to figure out what information is included in statuskit-cloud.db. Specifically the following tables:

ZINVITEDUSER
ZRECEIVEDINVITATION

There are phone numbers and emails within the tables, including timestamps. I'm assuming these are iMessages/sms texts but the timestamps seem to be a bit random and it doesn't include all texts.

Can anyone tell me what this information under StatusKit is/where it's coming from?

**Comment by scottfurry** [comment]
A path starting with a tilde, i.e. `~/` is Unix/Linux shorthand for a user's home directory. On MacOS, `/[USER]/Library` path is where user installed applications place application needed files or use as information storage. Somewhere on your Mac you have an application that uses the `StatusKit` name (possibly this - https://statuskit.com/).

As for the specifics of the tables, that's up to the makers of the software. From description, my guess is this is some kind of contact information. Your best bet is to contact makers of software to understand what they've placed *under the hood* of the database.
